cloudrail/knowledge/rules/rule_metadata.py

```
@@ -85,7 +85,13 @@ def rule_matches_query(rule_id: str, rule_name: str, query: Optional[str]) -> bo
 
 
 def get_rule_metadata_content(provider: CloudProvider) -> dict:
-    provider_name = 'aws' if provider == CloudProvider.AMAZON_WEB_SERVICES else 'azure'
+    if provider == CloudProvider.AMAZON_WEB_SERVICES:
```

I would prefer to throw an exception if the provider is missing and don't default to GCP

---

@Tomerva what do you think about this?

---

i can add to CloudProvider this method:

`    def to_shorthand_string(self):
        if self == CloudProvider.AMAZON_WEB_SERVICES:
            return 'aws'
        return self.name.lower()`

and then: provider_name = provider.to_shorthand_string()

WDYT?

---

looks better. Just add tests